index.js

```diff
@@ -50,6 +54,20 @@ module.exports = {
           .catch(this._errorMessage.bind(this));
       },
 
+      _tagVersion: function(version) {
+        var tagVariable = this.readConfig('tagVariable');
+
+        if (!tagVariable) { return; }
```

Comment:
should you throw a `log.warn` here?

Reply:
this was intended as an optional feature, but I agree we should have a log here. In fact I was thinking of making a separate ember addon to complete this index.html processing job. But let us do not spend too much time on this